rootfs/etc/nginx/lua/update.lua

```diff
@@ -0,0 +1,9 @@
+if ngx.var.request_method == "POST" then
+    ngx.req.read_body()
+    local backends_json = ngx.req.get_body_data()
+    local shared_memory = ngx.shared.shared_memory
+    shared_memory:set("CFGS", backends_json, 0)
```

Comment:
what if setting to the dictionary fails? We still return 200 and controller will think everything went well.